BUG-1230 standalone postgresql v12-->v13

DavidVadnais opened the pull request:
Fixes #1230

## Summary
This PR addresses the startup issues with the docker standalone startup.

## Testing Performed
I manually tested the following scenarios:

*  Running the application
*  Creating a superuser account
*  Creating a queue
*  Creating and deleting a ticket

## Known Issue: Migration Handling
One limitation of this fix is that it does not handle database migrations automatically. If the existing PostgreSQL database was initialized with version 12 or older, attempting to open the volume with a PostgreSQL 13 container will result in an error due to version incompatibility.

## Next Steps
A proper migration strategy (e.g., using [pg_upgrade](https://www.postgresql.org/docs/13/pgupgrade.html) or a documented backup/restore process) may be required for upgrades.

Do you need me to make these changes or is this PR fine as it is?

DavidVadnais commented:
Additionally, we could upgrade all the way to PostgreSQL 17 rather then just going up by one level.

uhurusurfa commented:
Yes, makes sense to use the very latest version but use an nevironment variable to set the version - something like this perhaps:
`image: postgres:${POSTGRES_MAJOR_VERSION:-17}-bullseye

Then add a line to the docker readme indicating the use of the environment variable.

DavidVadnais commented:
Made the suggested changes and I am testing them now. 

Also I wanted to upgrade the yml version (mentioned in #1200) so I did. 
First I bumped the `version: '2'` to `version: '3'` this gave me a deprecation warning. I found that "the version field in docker-compose.yml is no longer required and is now considered obsolete." - [source: this docker-hub post](https://forums.docker.com/t/docker-compose-yml-version-is-obsolete/141313).

DavidVadnais commented:
I wiped the volumes and then brought up a new set.

```
docker system prune
docker volume rm $(docker volume ls -q)
docker compose up
```

Then on this fresh set of volumes I created a new super user, added a queue, created a ticket, and deleted a ticket.

uhurusurfa reviewed (changes requested):
Looks good to me code-wise.

Can you add an additional step to the docs/standalone.rst file in the "Installation using docker-compose" section where step 3 would be something like:

3. Set the POSTGRES major version if you are deploying to an exsisting POSTGRES instance other than the default shown within the docker-compose file: 
    export POSTGRES_MAJOR_VERSION=15

uhurusurfa commented:
Also pleasse change the documentation to use:
` docker compose up`

... instead of the deprecated :
` docker-compose up`

DavidVadnais commented:
I made the requested docs changes but I have to run.  Ill test the docs to make sure the look right in 1-3 hours.

uhurusurfa reviewed (approved):
👍 Checked the formatting of the docs change and it looks good.
